packages/vsce/src/utils/resourceUtils.ts

```diff
@@ -61,6 +61,7 @@ export async function runGetResource({
   }
 
   // Making a second attempt as ltpa token has expired
+  CICSLogger.debug("Retrying as validation of the LTPA token failed because the token has expired.");
```

Comment:
Wonder if these messages should be constants, seeing as you added that messages file.

```suggestion
  CICSLogger.debug("Validation of the LTPA token failed because the token has expired. Retrying with basic authentication.");
```